ab3d2_source/bss/io_bss.s

```diff
@@ -28,4 +28,5 @@ io_ObjectName_vb:		ds.b	160
 io_Buffer_vb:			ds.b	80   ; todo - can these be merged ?
 
 ; File info block
+_io_FileInfoBlock::
```

Comment:
The dosheaders say that FileInfoBlock has to be 4byte aligned. This would be a good opportunity to enforce it here.

Reply:
I am pretty sure it already is - if you check the io_bss.s file it begins with an align 4 directive and contains only longs and aligned blocks of bytes (160 and 80 respectively) after that, followed immediately by the FIB.